drop_server/backend/views.py

```diff
@@ -107,6 +110,10 @@ def get_if_modified_since(request):
     return (False, datetime.fromtimestamp(0)) if since is None else (True, dateparser.parse(since))
 
 
+def set_last_modified(response, modificationDate):
+    response.headers['Last-Modified'] = modificationDate.strftime("%a, %d %b %Y %H:%M:%S GMT")
```

Comment:
%Z is the time zone name and %z is the time zone offset. The name however would not be GMT but UTC.

See https://docs.python.org/3/library/datetime.html?highlight=datetime#strftime-strptime-behavior